addons/website/views/website_templates.xml

```diff
@@ -326,7 +326,7 @@
             <!-- Brand -->
```

Comment:
Commit message:
In my previous review, I made you change the formulation "navbar mobile menu toggler" to "mobile menu navbar toggler" because it is the toggler of the navbar, which you did; but you also need to change it in the commit title :wink: Because at first when reading the title, I thought we were talking about the phone icon that toggles the mobile preview :grimacing: 
=> `[FIX] website: hide the overlay on the mobile menu navbar toggler`